Sources/XCRemoteCache/Network/NetworkClientImpl.swift

```diff
@@ -125,7 +130,7 @@ class NetworkClientImpl: NetworkClient {
         dataTask.resume()
     }
 
-    private func makeDownloadRequest(_ request: URLRequest, output: URL, completion: @escaping (Result<Void, NetworkClientError>) -> Void) {
+    private func makeDownloadRequest(_ request: URLRequest, output: URL, retries: Int, completion: @escaping (Result<Void, NetworkClientError>) -> Void) {
```

Comment:
Can we add a unit test for this logic as we have for upload?

Reply:
Done